test_linters.py

```diff
@@ -0,0 +1,11 @@
+# Función para calcular la suma de los primeros n números naturales
```

Comment:
I don't understand the reason for having this file. How does this file play with a Django application?